easybuild/easyconfigs/s/scikit-build/scikit-build-0.11.1-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonBundle'
+
+name = 'scikit-build'
+version = '0.11.1'
+
+homepage = 'https://scikit-build.readthedocs.io/en/latest'
+description = """Scikit-Build, or skbuild, is an improved build system generator
+for CPython C/C++/Fortran/Cython extensions."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+builddependencies = [
+    ('binutils', '2.36.1'),
```

Comment:
@Micket Weren't we talking about changing this to use `GCC` instead? Or is that not agreed yet?
I remember there was a suggestion of avoiding this annoying `binutils` dep...

Reply:
well we are already doing the good old binutils 2.36.1 in 172 other easyconfigs for GCCcore 10.3.0, so I didn't want to make 1 exception to that pattern.

Reply:
Fair enough.